nativeauthenticator/templates/native-login.html

```diff
@@ -33,6 +33,7 @@
             {% endif %}
             <label for="username_input">Username:</label>
             <input id="username_input" type="text" autocapitalize="off" autocorrect="off" class="form-control" name="username" val="{{username}}" tabindex="1" autofocus="autofocus" />
+            <p></p>
```

Comment:
Why the empty <p>?

Reply:
A hack in the design the give more space between things 😅 I can take it off